swisstopo_geoengine/geo_view/geo_raster_layer.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class GeoRasterLayer(models.Model):
+    _inherit = 'geoengine.raster.layer'
+
+    raster_type = fields.Selection(selection_add=[('swisstopo', 'Swisstopo')])
+    layername = fields.Char('Layer Machine Name')
+    #time_dimension = fields.Char('Time Dimension')
```

Comment:
To remove ?

Reply:
Actually I have forgotten to add this field, it is supposed to be used in order to set the time dimension in the swisstopo URLs. It defaults to ``current``:
https://github.com/OCA/geospatial/blob/7f66a4e00fe1c30502d0f0efc23fdd04987ccec6/swisstopo_geoengine/static/src/js/swisstopo_geoengine.js#L15